src/sinks/aws_s3/config.rs

```diff
@@ -82,7 +76,7 @@ impl GenerateConfig for S3SinkConfig {
             filename_extension: None,
             options: S3Options::default(),
             region: RegionOrEndpoint::default(),
-            encoding: EncodingConfig::from(StandardEncodings::Text).into(),
+            encoding: (None::<FramingConfig>, TextSerializerConfig::new()).into(),
```

Comment:
I think I tried that slightly different and I got a weird `rustc` error that this is not allowed because I already implemented it for the tuple. Let me check again

Reply:
There it is:

```
error[E0119]: conflicting implementations of trait `std::convert::From<(std::option::Option<_>, _)>` for type `codecs::encoding::config::EncodingConfigWithFraming`
   --> src/codecs/encoding/config.rs:141:1
    |
128 | / impl<F, S> From<(Option<F>, S)> for EncodingConfigWithFraming
129 | | where
130 | |     F: Into<FramingConfig>,
131 | |     S: Into<SerializerConfig>,
...   |
138 | |     }
139 | | }
    | |_- first implementation here
140 | 
141 | / impl<S> From<S> for EncodingConfigWithFraming
142 | | where
143 | |     S: Into<SerializerConfig>,
144 | | {
...   |
150 | |     }
151 | | }
    | |_^ conflicting implementation for `codecs::encoding::config::EncodingConfigWithFraming`
    |
    = note: upstream crates may add a new impl of trait `std::convert::From<(std::option::Option<_>, _)>` for type `codecs::encoding::SerializerConfig` in future versions
```

😐

Reply:
Oh right, lack of minimum specialization. Argh. Okay. 😭